t/re-bug.t

```diff
@@ -0,0 +1,74 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+use lib 'lib';
+use Test::Nginx::Socket::Lua;
+use Cwd qw(cwd);
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 4);
+
+my $pwd = cwd();
+
+our $HttpConfig = <<_EOC_;
+    lua_package_path "$pwd/lib/?.lua;../lua-resty-lrucache/lib/?.lua;;";
+    init_by_lua_block {
+        require "resty.core"
+        ngx.re.match('c', 'test', 'jo')
+    }
+_EOC_
+
+no_long_string();
+check_accum_error_log();
+run_tests();
+
+__DATA__
+
+=== TEST 1: PCRE MAP_JIT bug on macOS
+--- skip_eval
+4: $^O ne 'darwin'
+--- http_config eval: $::HttpConfig
+--- config
+    location /re {
+        content_by_lua_block {
+            ngx.say(ngx.re.sub('c', 'a', 'b', ''))
+            ngx.say(ngx.re.sub('c', 'a', 'b', 'jo'))
+            ngx.say("it works!")
+        }
+    }
+--- request
+    GET /re
+--- response_body
+c0
+c0
+it works!
```

Comment:
The test is not only testing the error log, but also whether NGINX crashed or not. So the response was left in there since if NGINX crashed while executing the regex, the last line will not execute.

Reply:
The above reasoning is that if we get both `c0` lines, then we already know nginx did not crash

Reply:
That's fair enough. I have removed them from both tests.